neml/drivers.py

```diff
@@ -596,7 +596,212 @@ def strain_cyclic(model, emax, R, erate, ncycles, T = 300.0, nsteps = 50,
       "energy_density": np.array(ecycle), "plastic_work": np.array(pcycle),
       "history": driver.stored_int[-1], "time": np.array(time)}
 
-def strain_cyclic_followup(model, emax, R, erate, ncycles, 
+def strain_cyclic_extrapolated(model, emax, R, erate, ncycles, T = 300.0, nsteps = 50,
+    sdir = np.array([1,0,0,0,0,0]), hold_time = None, n_hold = 25,
+    verbose = False, check_dmg = False, dtol = 0.75, min_cycle=3, unit_extrapolate = 10,
+    jump_del_N=10,allowable_jump_stress=5.0):
+  """
+    Strain controlled cyclic test extrapolation.
+
+    Extra Keyword Args:
+      min_cycle                minimum cycles to start the extrapolation process
+      unit_extrapolate         number of cycles to perform single cycle extrapolation
+      jump_del_N               number of cycles to jump
```

Comment:
what does `del` means? `delta`? if so please rename as `jump_delta_N` or `jump_N `.
`jump_del_N` is confusing

Reply:
changed to delta